docs/src/start/send-thin-edge-data.md

```diff
@@ -24,21 +24,21 @@ A simple single-valued measurement like a temperature measurement, can be repres
 
 with the key-value pair representing the measurement type and the numeric value of the measurement.
 
-This measurement can be sent from the Thin Edge device to the cloud by publishing this message to the `tedge/measurements` MQTT topic.
+This measurement can be sent from the Thin Edge device to the cloud by publishing this message to the `te/+/+/+/+/m/+` MQTT topic.
 Processes running on the Thin Edge device can publish messages to the local MQTT broker using any MQTT client or library.
 In this tutorial, we'll be using the `tedge mqtt pub` command line utility for demonstration purposes.
 
 The temperature measurement described above can be sent using the `tedge mqtt pub` command as follows:
 
-```sh te2mqtt
-tedge mqtt pub tedge/measurements '{"temperature": 25}'
+```sh te2mqtt formats="v1"
+tedge mqtt pub te/device/main///m/environment '{"temperature": 25}'
```

Comment:
The `environment` type is used by the example but not introduced at all.

Reply:
Actually on careful reconsideration I added a note here cb2eebe33c9d2893230a822cfb68cce3499b7332